src/test/java/gov/cabinetoffice/gap/adminbackend/controllers/GrantExportBatchControllerTest.java

```diff
@@ -97,4 +106,46 @@ void errorThrown() throws Exception {
 
     }
 
+    @Nested
+    class getExportBatchInfo {
+
+        @Test
+        void successfullyGetsBatchInfo() throws Exception {
+            UUID randomUUID = UUID.randomUUID();
+            final GrantExportBatchEntity grantExportBatchEntity = GrantExportBatchEntity.builder()
+                    .id(randomUUID)
+                    .applicationId(1)
+                    .createdBy(1)
+                    .build();
+            final GrantExportBatchDTO grantExportBatchDTO = GrantExportBatchDTO.builder()
+                    .exportBatchId(randomUUID)
+                    .applicationId(1)
+                    .createdBy(1)
+                    .status(GrantExportStatus.COMPLETE)
+                    .emailAddress("test123@test.com")
+                    .created(Instant.now())
+                    .lastUpdated(Instant.now())
+                    .location("here")
+                    .build();
+
+            when(grantExportBatchService.getGrantExportBatch(mockExportId))
+                    .thenReturn(grantExportBatchEntity);
+            when(grantExportMapper.grantExportBatchEntityToGrantExportBatchDTO(grantExportBatchEntity)).thenReturn(grantExportBatchDTO);
+
+            mockMvc.perform(get("/grant-export-batch/" + mockExportId).header(HttpHeaders.AUTHORIZATION,
+                            LAMBDA_AUTH_HEADER)).andExpect(status().isOk())
```

Comment:
We don't need to include the header setting here because we are not adding a lambda auth header to the request being sent to this endpoint since the lambda isn't isn't calling this endpoint